src/__experimental__/components/numeral-date/numeral-date.stories.mdx

```diff
@@ -141,6 +141,22 @@ const MyComponent = () => (
   </Story>
 </Preview>
 
+### Enabling the adaptive behaviour
+The inline label can change to be top aligned at a breakpoint. Enable this by passing in a number to the `adaptiveLabelBreakpoint` prop. This corresponds to a px screen width
+
+<Preview>
+  <Story name="enabling adaptive behaviour">
+    <NumeralDate 
+      dateFormat={['dd','mm','yyyy']} 
+      label="Inline"
+      labelInline
```

Comment:
```suggestion
```
based on previous suggestion to remove `labelInline` from condition.

Reply:
Then this story wouldn't be showing the adaptive behaviour at all. It need to have an inline label to show it changing to top aligned at the breakpoint